model_compression_toolkit/core/common/quantization/quantization_params_fn_selection.py

```diff
@@ -14,6 +14,7 @@
 # ==============================================================================
 
```

Comment:
I see that here it is also "fn_selection" (so probably that is why you called the new one "selection" as well)
Still think it's confusing since this file actually returns a function for "parameters selection".
So, if you decide to change the name of the other file, maybe change this one too.